SPECS/libguestfs/libguestfs.spec

```diff
@@ -747,19 +741,17 @@ fi
 mv README README.orig
 sed 's/@VERSION@/%{version}/g' < %{SOURCE4} > README
 
-# Re-enable upstream cache and local repos
-mkdir -pv %{_sysconfdir}/yum.repos.d
-cp %{SOURCE9} %{_sysconfdir}/yum.repos.d/allrepos.repo
-
 # Download appliance, since our chroot TDNF config does not have `keepcache=1`
 # Must keep in sync with BRs under "Build requirements for the appliance"
 # Download to
 mkdir -pv %{_var}/cache/tdnf
-tdnf install --downloadonly -y --disablerepo=* \
-  --enablerepo=local-repo \
-  --enablerepo=upstream-cache-repo \
-  --enablerepo=toolchain-repo \
-  --alldeps --downloaddir %{_var}/cache/tdnf \
+tdnf repolist
+tdnf install --downloadonly -y \
+    --disablerepo=* \
+    --enablerepo=local-repo \
+    --enablerepo=toolchain-repo \
+    --enablerepo=upstream-cache-repo \
+    --alldeps --downloaddir %{_var}/cache/tdnf \
     acl \
     attr \
```

Comment:
We should add `ocaml-augeas-devel` to be installed here with tdnf to keep the list up to date

Reply:
Added it in alphabetic order as well